ktor-server/ktor-server-core/common/src/io/ktor/server/config/MapApplicationConfig.kt

```diff
@@ -196,8 +195,8 @@ internal class MapApplicationConfigValue(
     }
 }
 
-private fun combine(root: String, relative: Any): String =
-    if (root.isEmpty()) relative.toString() else "$root.$relative"
+private fun combine(root: String, relative: Int): String = combine(root, relative.toString())
+private fun combine(root: String, relative: String): String = if (root.isEmpty()) relative else "$root.$relative"
```

Comment:
Investigating why changing the signature from `combine(String, String)` to `combine(String, Any)` caused ProGuard warning...